src/Database/Adapter/MariaDB.php

```diff
@@ -1635,6 +1635,61 @@ public function deleteDocument(string $collection, string $id): bool
         return $deleted;
     }
 
+    /**
+     * Delete Documents
+     *
+     * @param string $collection
+     * @param array<\Utopia\Database\Query> $queries
+     *
+     * @return bool
+     */
+    public function deleteDocuments(string $collection, array $queries): bool
+    {
+        $name = $this->filter($collection);
+        $where = [];
+
+        $queries = array_map(fn ($query) => clone $query, $queries);
+
+        $conditions = $this->getSQLConditions($queries);
+        if(!empty($conditions)) {
+            $where[] = $conditions;
+        }
+
+        if ($this->sharedTables) {
+            $where[] = "table_main._tenant = :_tenant";
+        }
+
+        $sqlWhere = !empty($where) ? 'WHERE ' . implode(' AND ', $where) : '';
+
+        $selections = $this->getAttributeSelections($queries);
+
+        $sql = "
+            USE {$this->database};
+            DELETE {$this->getAttributeProjection($selections, 'table_main')}
+            FROM {$this->getSQLTable($name)} as table_main
+            {$sqlWhere};
+        ";
```

Comment:
Since we are always doing a single table delete here:

```suggestion
        $sql = "
            DELETE FROM {$this->getSQLTable($name)}
            {$sqlWhere};
        ";
```